templates/components/news-banner/news-banner.twig

```diff
@@ -12,8 +12,8 @@
  * - news_banner_topic: Topic selected. Dynamically altered vi news-banner.js
  */
 #}
-<div {{ attributes }} class="su-news-banner">
-    {{ news_banner_image }}
-    {{ news_banner_title }}
-    {{ news_banner_topic }}
+<div {{ attributes }} class="su-news-banner {{ modifier_class }}">
```

Comment:
@jenbreese @boznik 

I have added the `modifier_class` here because that is the convention we have coming from Decanter but thinking about it now that is probably not necessary. We do use the modifier class for the variants but having both the `{{ attributes }}` variable and other HTML properties like class or ID hard wired on the component can produce duplicate properties. This would be better if it was just the `{{ attributes }}` variable on the outer tag and all css classes were added with `{{ attributes.addClass() }}`

Reply:
@jenbreese assigned.